integrated_channels/blackboard/exporters/content_metadata.py

```diff
@@ -34,3 +34,10 @@ def transform_enrollment_url(self, content_metadata_item):
         enrollment_url = content_metadata_item.get('enrollment_url', None)
         url_link = self.DESCRIPTION_TEXT_TEMPLATE.format(enrollment_url=enrollment_url)
         return url_link
+
+    def transform_course_unique_key(self, content_metadata_item):
+        """
+        Blackboard courseId cannot contain some chars used in edX courseIds, so using hash
+        We can always look up courses by externalId whch is the same as edX courseId
+        """
+        return content_metadata_item.get('key').replace('+','').replace(':','').replace('/','')
```

Comment:
I feel like there's gotta be a simpler way then 3 .replace

Reply:
replaced with client cdoe of hash instead...